crates/sui-json/src/tests.rs

```diff
@@ -405,8 +410,15 @@ fn test_basic_args_linter_top_level() {
         .build(path)
         .unwrap()
         .into_modules();
-    let example_package =
-        Object::new_package_for_testing(compiled_modules, TransactionDigest::genesis()).unwrap();
+    // SAFETY: genesis packages should never exceed max size
+    let (std_move_pkg, sui_move_pkg) =
```

Comment:
We actually don't impose the size limit for them when we create htem -- i.e. they get a size limit of `u64::MAX`, so you can probably get rid of the size parameter here.